src/ui/public/courier/search_source/index.js

```diff
@@ -18,6 +18,9 @@
  */
 
 export { SearchSourceProvider } from './search_source';
-export { migrateFilter } from './migrate_filter';
-export { decorateQuery } from './decorate_query';
-export { buildQueryFromFilters, luceneStringToDsl } from './build_query';
+export {
+  migrateFilter,
+  decorateQuery,
+  buildQueryFromFilters,
+  luceneStringToDsl
+} from '../../../../utils/es_query';
```

Comment:
Do we even need to export these from here now are can existing users of this module just import the functions from the new directory?

Reply:
We could, it just seemed like fewer changes to do it this way rather than go through each of the imports and change them. I'm open to changing this if you think it makes sense.

Reply:
Maybe I missed some, but it looked to me like the imports had already been changed. Intellij's refactoring tools should handle it. I'd just rip the band aid off cleanly so folks in the future aren't confused about where they should be importing from.

Reply:
Made the change. Wasn't even bad at all